### ISSUE TITLE
Unable to adjust font size in macOS Catalina 10.15.0

### ISSUE THREAD

**Issue by li-kbotelho** [dessibelle/Blue-Screen-Saver#6]

**Comment by dessibelle** [comment]
I'll look into this once I find the time to upgrade ;)

**Comment by dessibelle** [comment]
Looks like the `isPreview` parameter supplied in the `NSScreenSaverView initWithFrame:isPreview:` constructor is always true, and the font size calculation defaulted to using the preview font size of `6.0`, regardless of the setting.

Reversing the ternary if-assignment should be a simple fix, but it will instead make the preview look odd. I'll see if I can find a fix, or perhaps just calculate the preview font size based on frame size.

**Comment by dessibelle** [comment]
For all I can tell ScreenSaverEngine is calling `initWithFrame:isPreview:` with `isPreview` set to true all the time. I've filed a bug report with Apple (`FB7384145`). 

For the time being I have just disabled the preview logic, so actual screen saver rendering looks normal and the preview looks gigantic. Perhaps I can revert that later on. There's a new version hosted on the link in the README now.
